docs/_markdown/ark.analysis.md

```diff
@@ -0,0 +1,36 @@
+analysis
+====================
+
+These libraries are like ground zero. They are the base libraries to use for analysis.
```

Comment:
Maybe something more like 'These are the base libraries to use for post-segmentation analysis'?

Reply:
Yeah, I like that 😀.